view/src/main/java/cmm/apps/esmorga/view/eventlist/model/EventListUiState.kt

```diff
@@ -1,11 +1,15 @@
 package cmm.apps.esmorga.view.eventlist.model
 
+import cmm.apps.esmorga.domain.event.model.Event
+import kotlinx.serialization.Serializable
+
 data class EventListUiState(
     val loading: Boolean = false,
     val eventList: List<EventListUiModel> = emptyList(),
     val error: String? = null
 )
 
+@Serializable
```

Comment:
Is @Serializable needed in this class?

Reply:
The @serializable is needed for navigation to pass complex data in arguments

Reply:
But you use Event not EventListUiState right?

Reply:
what about this? you did not answer last question

Reply:
sorry ... I was confused about what @MiguelPerezAccenture was asking, he is right, this is not  need it